ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -160,6 +164,45 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-openstack-etcd-scaling
+- as: e2e-openstack-masterscale
+  minimum_interval: 240h
+  steps:
+    cluster_profile: openstack-vexxhost
+    test:
+    - as: openstack-test-cpms
+      commands: |
+        dir=$GOPATH/src/github.com/openshift/cluster-control-plane-machine-set-operator
+        git clone https://github.com/openshift/cluster-control-plane-machine-set-operator --branch release-4.14 --depth=1 $dir
+        export OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE="$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")"
+        cd $dir
+        make e2e-presubmit
+      from: stolostron
+      resources:
+        requests:
+          cpu: 100m
+    - ref: openstack-test-openstack
```

Comment:
@pierreprinetti for the periodic job, what would you like to be running after CCPMSO e2e, openstack-test only? or also regular e2e-openshift as well?